src/plugins/intel_cpu/src/nodes/experimental_detectron_roifeatureextractor.cpp

```diff
@@ -346,66 +183,244 @@ void ExperimentalDetectronROIFeatureExtractor::initSupportedPrimitiveDescriptors
     if (!supportedPrimitiveDescriptors.empty())
         return;
 
+    Precision inputPrec = getOriginalInputPrecisionAtPort(INPUT_FEATURES_START);
+    Precision outputPrec = getOriginalOutputPrecisionAtPort(OUTPUT_ROI_FEATURES);
+
+    if (inputPrec != Precision::FP32 || outputPrec != Precision::FP32) {
+        if ((outputPrec == Precision::BF16 || inputPrec == Precision::BF16) && mayiuse(avx512_core)) {
+            outputPrec = inputPrec = Precision::BF16;
+        } else {
+            outputPrec = inputPrec = Precision::FP32;
+        }
+    }
+
     std::vector<PortConfigurator> inDataConf;
     inDataConf.reserve(inputShapes.size());
-    for (int i = 0; i < inputShapes.size(); ++i)
-        inDataConf.emplace_back(LayoutType::ncsp, Precision::FP32);
+    inDataConf.emplace_back(LayoutType::ncsp, Precision::FP32);
+    for (int i = 1; i < inputShapes.size(); ++i)
+        inDataConf.emplace_back(LayoutType::ncsp, inputPrec);
```

Comment:
why not include other layouts?

Reply:
I think it should be done in separate task